docs/rules/no-confusing-arrow.md

```diff
@@ -31,8 +32,34 @@ The following patterns are not considered warnings:
 /*eslint no-confusing-arrow: 2*/
 /*eslint-env es6*/
 
-var x = a => { return 1 ? 2 : 3; }
-var x = (a) => { return 1 ? 2 : 3; }
+var x = a => { return 1 ? 2 : 3; };
+var x = (a) => { return 1 ? 2 : 3; };
+```
+
+## Options
+
+This rule accepts a single options argument with the following defaults:
+
+```json
+{
+    "rules": {
+        "no-confusing-arrow": [2, {"allowParens": false}]
+    }
+}
+```
+
+`allowParens` is a boolean setting that can be `true` or `false`:
+
+1. `true` relaxes the rule and accepts parenthesis as a valid "confusion-preventing" syntax.
+2. `false` warns even if the expression is wrapped in parenthesis
+
+When `allowParens` is set to `true` following patterns are no longer considered as warnings:
```

Comment:
Should we separate two things? Looks like @pedrottimark haven't yet came to `no-confusing-arrow` so I'm not breaking anything. I'd prefer we continue changing these through his patches and keep this one specific to the problem at hand.